core/chains/evm/client/node_lifecycle.go

```diff
@@ -120,7 +120,7 @@ func (n *node) aliveLoop() {
 		lggr.Debug("Polling disabled")
 	}
 
-	var latestReceivedBlockNumber int64 = -1
+	var highestReceivedBlockNumber int64 = n.LatestReceivedBlockNumber()
```

Comment:
Can this be zero on a newly initialised node? New nodes ought to come with this set to -1 so that if you receive a literal block with number 0, it registers as one higher.

Reply:
Fixed